demo-notebooks/guided-demos/3_basic_interactive.ipynb

```diff
@@ -55,20 +57,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Create and configure our cluster object (and appwrapper)\n",
+    "# Create and configure our cluster object\n",
+    "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\"\n",
```

Comment:
same question about specifying local queue/using annotation - applies everywhere this is mentioned